substrate/client/network/src/protocol/notifications/behaviour.rs

```diff
@@ -1423,7 +1423,6 @@ impl NetworkBehaviour for Notifications {
 									let delay_id = self.next_delay_id;
 									self.next_delay_id.0 += 1;
 									let delay = futures_timer::Delay::new(ban_duration);
-									let peer_id = peer_id;
```

Comment:
wonder why these were here before?

Reply:
it used to be `let peer_id = *peer_id;` until [this commit](https://github.com/paritytech/substrate/commit/4617267aa82f), which just removed the `*` and left the remaining no-op line around.